src/nxt_h1proto.c

```diff
@@ -884,6 +882,34 @@ nxt_h1p_request_body_read(nxt_task_t *task, nxt_http_request_t *r)
         break;
     }
 
+    if (r->chunked) {
+        h1p->chunked_parse.mem_pool = r->mem_pool;
+        in = h1p->conn->read;
+        size = nxt_buf_mem_used_size(&in->mem);
+        if (size == 0) {
+            status = NXT_HTTP_NOT_IMPLEMENTED;
+            goto error;
+        }
+
+        nxt_buf_t *out = nxt_http_chunk_parse(task, &h1p->chunked_parse, in);
+
+        if (h1p->chunked_parse.chunk_error ||
```

Comment:
Is this conditional meant to be formatted like this?

Reply:
No, it looks simply like a snafu...